arm-resources/resources/2016-07-01/swagger/resources.json

```diff
@@ -0,0 +1,2459 @@
+            "Incremental",
+            "Complete"
+          ],
+          "x-ms-enum": {
+            "name": "DeploymentMode",
+            "modelAsString": false
+          }
+        },
+        "debugSetting": {
+          "$ref": "#/definitions/DebugSetting",
+          "description": "The debug setting of the deployment."
+        }
+      },
+      "required": [
+        "mode"
+      ],
+      "description": "Deployment properties."
+    },
+    "DebugSetting": {
+      "properties": {
+        "detailLevel": {
+          "x-ms-client-flatten": true,
+          "type": "string",
+          "description": "The debug detail level."
+        }
+      }
+    },
+    "Deployment": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DeploymentProperties",
+          "description": "The deployment properties."
+        }
+      },
+      "description": "Deployment operation parameters."
+    },
+    "DeploymentExportResult": {
+      "properties": {
+        "template": {
+          "type": "object",
+          "description": "The template content."
+        }
+      },
+      "description": "The deployment export result. "
+    },
+    "ResourceManagementErrorWithDetails": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "The error code returned from the server."
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message returned from the server."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the error."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceManagementErrorWithDetails"
+          },
+          "description": "Validation error."
+        }
+      },
+      "required": [
+        "code",
+        "message"
+      ]
+    },
+    "AliasPathType": {
+      "properties": {
+        "path": {
+          "type": "string",
+          "description": "The path of an alias."
+        },
+        "apiVersions": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The api versions."
+        }
+      },
+      "description": "The type of alias paths. "
+    },
+    "AliasType": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The alias name."
+        },
+        "paths": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AliasPathType"
+          },
+          "description": "The paths for an alias."
+        }
+      },
+      "description": "The alias type. "
+    },
+    "ZoneMappingType": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The location of this zone mapping."
+        },
+        "zones": {
+          "type": "array",
+          "items": {
```

Comment:
Do we need it in "items"? As I see we have "description" for "zones" (save level as "items").